core/services/assets/AssetManager.php

```diff
@@ -156,7 +156,7 @@ public function addVendorJavascript(
         $load_in_footer = true,
         $version = ''
     ) {
-        $dev_suffix = wp_scripts_get_suffix('dev');
```

Comment:
`dev` works only for WP versions with `$wp_version` ending with `-src`. More [here](https://developer.wordpress.org/reference/functions/wp_scripts_get_suffix/)